pages/[id].js

```diff
@@ -1,5 +1,109 @@
-import ActivityCardDetails from "@/components/ActivityCardDetails";
+import styled from "styled-components";
+import Image from "next/image";
+import { activities } from "@/lib/dummydata";
+import { useRouter } from "next/router";
+import Link from "next/link";
 
-export default function JokeDetailsPage() {
-  return <ActivityCardDetails />;
-}
\ No newline at end of file
+const StyledHeadline = styled.h2`
+  text-align: center;
+`;
+
+const StyledButton = styled.button`
+
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: black;
+  position: absolute;
+  left: 8%;
+  padding: 1% 3%;
```

Comment:
percentage is better than vw for padding, but the best practice is still using flex/grid + rem or px values. Please consider this in the future. For now its fine.